index.html

```diff
@@ -1308,6 +1325,12 @@ <h2 id="privacy-considerations">
         A user agent SHOULD provide a means for the user to review, update, and reset the
         [=permission=] [=permission/state=] of [=powerful features=] associated with an [=origin=].
       </p>
+      <p>
+        Some [=user agents=] support <dfn data-lt="administrator">administrators</dfn> who can
```

Comment:
I expect the `<dfn>` to move somewhere else, perhaps Infra, in the future, but I think the discussion about putting it there will be easier if it's already used in a spec or two first.